sky/resources.py

```diff
@@ -37,6 +42,22 @@
     'gpus': 'accelerators',
 }
 
+MEMORY_SIZE_UNITS = {
+    'b': 1,
+    'k': 2**10,
+    'kb': 2**10,
+    'm': 2**20,
+    'mb': 2**20,
+    'g': 2**30,
+    'gb': 2**30,
+    't': 2**40,
+    'tb': 2**40,
+    'p': 2**50,
+    'pb': 2**50,
+}
+
+ACCELERATORS_PATH = 'common/metadata.csv'
```

Comment:
I see this depends on https://github.com/skypilot-org/skypilot-catalog/pull/134 so we must make sure that's merged first.